libs/libc/modlib/modlib_symbols.c

```diff
@@ -55,6 +56,11 @@ struct mod_exportinfo_s
   FAR const struct symtab_s *symbol; /* Symbol info returned (if found) */
 };
 
+typedef struct {
+	uint8_t	*epName;	/* Name of global symbol */
+	void	*epAddr;	/* Address of global symbol */
+} epTable_t;
+
```

Comment:
Done, but what's the rationale?

Reply:
I think the issue is because the typedef makes it more difficult to know what is that type (more opaque) and Hungarian notation (like used by FreeRTOS) is not good visually. I could be wrong here, maybe others can confirm. @xiaoxiang781216 @gregory-nutt @jerpelea ?